c2cgeoportal/tests/test_raster.py

```diff
@@ -77,11 +77,11 @@ def test_raster(self):
 
     def test_absolute_path(self):
         from c2cgeoportal.lib.raster.georaster import GeoRaster
-        gr = GeoRaster("c2cgeoportal/tests/data/dem_absolute.shp")
+        gr = GeoRaster("/src/c2cgeoportal/tests/data/dem_absolute.shp")
         tile = gr._get_tile(548000, 216000)
-        self.assertEqual(tile.filename,
-                         "/home/sbrunner/regiogis/regiogis/c2cgeoportal/"
-                         "c2cgeoportal/tests/data/dem.bt")
+        self.assertEqual(
+            tile.filename,
+            "/home/sbrunner/regiogis/regiogis/c2cgeoportal/c2cgeoportal/tests/data/dem.bt")
```

Comment:
Yes because he is present in the `c2cgeoportal/tests/data/dem_absolute.shp` file :-)